hs_sitemap/templates/sitemap.html

```diff
@@ -0,0 +1,28 @@
+{% extends "base.html" %}
+
+{% block all_content %}
+    <style>
+    h1,h2,h3,h4 {
+        margin: 20px;
+    }
+    h3 {
+        padding-left: 20px;
+    }
+    h4 {
+        padding-left: 40px;
+    }
+    </style>
```

Comment:
Simply wasn't a reason not to embed them for styles that are not applicable to any other pages, especially when this page serves a very specific, narrow case and is meant chiefly to be consumed by machines.